x-pack/plugins/ml/public/application/services/ml_api_service/data_frame_analytics.ts

```diff
@@ -175,4 +175,12 @@ export const dataFrameAnalytics = {
       body,
     });
   },
+  analyticsFields(indexPatternTitle: string, isRollup: boolean = false) {
+    const query = isRollup === true ? { rollup: true } : {};
+    return http<any>({
```

Comment:
Can the use of `any` be avoided here?

Reply:
This response type should be the same as `/jobs/new_job_caps`

Reply:
Updated both in fa3e81bdcbfd5b0fe270287619220c7907a36a59